core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.spec.ts

```diff
@@ -338,6 +338,110 @@ describe('DotEmaComponent', () => {
                 });
             });
 
+            describe('create', () => {
+                it('should open a dialog, trigger a save from the store and send a post message when saving a contentlet', (done) => {
```

Comment:
Isn't this tests doing wayyy too much?

Reply:
Yes they are, this flow is reallyyyyy long